Automattic/wp-calypso
# Logged in Performance Profiler: Update page selector to include search icon

Related to https://github.com/Automattic/dotcom-forge/issues/9246

## Proposed Changes

* Include a search icon in the page selector to match the design
* Disable the device control when loading
![image](https://github.com/user-attachments/assets/fcc7d1de-799f-4283-8ccf-45a9623224ee)


## Why are these changes being made?


* Polish to closely match design

## Testing Instructions



* Go to s`sites/performance/site`
* Verify design

## Pre-merge Checklist
